core/src/main/java/com/adobe/aio/workspace/Workspace.java

```diff
@@ -270,6 +280,10 @@ public Builder addMetascope(final String metascope) {
         jwtbuilder = JwtContext.builder();
       }
       jwtbuilder.addMetascope(metascope);
+      if(oAuthBuilder == null){
```

Comment:
this `addMetascope()` was only kept for jwt backward compatibility, let's not use it to set the oautscope ...

Reply:
please remove these new lines this only to be used for JWT

Reply:
let's remove line 275-277

Reply:
Review Comments Incorporated